### ISSUE TITLE
A11y support

### ISSUE THREAD

**Issue by timove** [Polymer/shop#45]
I use Chrome (51.0.2704.84) and ChromeVox (50.0.2661.83). Most of the usual a11y features of Polymer and its webcomponents seem to be missing in the shop app. Is this the consequence of implementing pages as webcomponents? Since this is a case study: can this be fixed? How?

**Comment by blasten** [comment]
@timove is there an issue in particular that you ran into? Please provide concrete examples and steps to reproduce them.

**Comment by timove** [comment]
@blasten Using ChromeVox on https://shop.polymer-project.org/ the only thing that's audible is "shop" for the whole page. 

I cannot navigate the page using ChromeVox using e. g. "next sentence" operations. 

When navigating via tab, ChromeVox remains silent, e.g. when focussing paper-tabs. In contrast to using the paper-tabs element's demo (https://elements.polymer-project.org/elements/paper-tabs?view=demo:demo/index.html&active=paper-tabs), where ChromeVox reads focussed items, etc..

**Comment by timove** [comment]
@blasten ChromeVox is the only screen reader I've tried so far. I chose it, because of the recommendations in various videos by the Polymer team. Maybe my findings are just shortcomings of ChromeVox?

Looking at https://www.w3.org/TR/shadow-dom/#assistive-technology: "User agents with assistive technology traverse the flat tree, and thus enable full use of WAI-ARIA [WAI-ARIA] semantics in the shadow trees."

**Comment by timove** [comment]
@blasten I tried other screen readers: iOS 9 VoiceOver on an iPhone and NVDA for Windows. It turns out, each of those three behaves differently. VoiceOver and NVDA work better with respect to the issues I observed with ChromeVox: both can read the shop application well. So I guess this is a shortcoming of ChromeVox and suggest to close this issue.

**Comment by frankiefu** [comment]
Yeah, ChromeVox extension is pretty broken in ShadowDOM.  There is nothing we can do until ChromeVox extension is fixed.
